### DISCUSSION TITLE
Custom government/org types

### DISCUSSION THREAD

**Discussion by 1ByteBit** [thatguycy/WorldDynamics-Engine#8] [Ideas]
The ability to create a custom type of selectable organization/government via a config file, alongside any properties that may have to be selected.

This will be particularly useful for servers which wish to use a custom naming scheme or add niche/specific organizations/governments

**Comment by stoleyourharvs** [comment]
Hi, I've committed this to the master branch, built and tested, I'll release a .jar file as 0.1.6 later.

**Reply by 1ByteBit** [reply]
Wow, you're fast!

**Reply by stoleyourharvs** [reply]
https://github.com/thatguycy/WorldDynamics-Engine/releases/tag/v0.1.6
